tests_e2e/tests/ext_sequencing/ext_seq_test_cases.py

```diff
@@ -0,0 +1,300 @@
+def add_one_dependent_ext_without_settings():
+    # Dependent extensions without settings should be enabled with dependencies
```

Comment:
could you add a short description to the other test cases as well? the name provides a good hint, but let's add a few lines about why the test case is testing, why it is interesting, and stuff like that